scipy/stats/_qmc.py

```diff
@@ -901,6 +903,17 @@ class LatinHypercube(QMCEngine):
         Dimension of the parameter space.
     centered : bool, optional
         Center the point within the multi-dimensional grid. Default is False.
+    optimization : {None, "random-CD"}, optional, default None
```

Comment:
```suggestion
    optimization : {None, "random-CD"}
```
> When a parameter can only assume one of a fixed set of values, those values can be listed in braces, with the default appearing first:
```
order : {'C', 'F', 'A'}
    Description of `order`.
```
See https://numpydoc.readthedocs.io/en/latest/format.html#parameters. 

Besides that, it should be _either_ `optional` with the default described in the text OR `default: None`, not both.

Reply:
Thanks I was not sure about the syntax here. I might put it in the text then to match the style of the file.